synapse/events/builder.py

```diff
@@ -98,7 +98,10 @@ def is_state(self):
         return self._state_key is not None
 
     async def build(
-        self, prev_event_ids: List[str], auth_event_ids: Optional[List[str]],
+        self,
+        prev_event_ids: List[str],
+        auth_event_ids: Optional[List[str]],
+        inherit_depth: bool = False,
```

Comment:
I think we should not have a default value here since all callers provide it anyway?

Reply:
We don't define `inherit_depth` at this caller for example,

https://github.com/matrix-org/synapse/blob/7950aa8a27c3f45184c96fda210c62d068dd2591/synapse/handlers/message.py#L1391-L1394

Reply:
Isn't this the only other caller though (sans tests)?

Reply:
Removed the default ⏩